src/Serilog.Extensions.Logging.File/Serilog.Extensions.Logging.File.csproj

```diff
@@ -1,10 +1,10 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <Description>Add file logging to ASP.NET Core apps with one line of code.</Description>
-    <VersionPrefix>2.0.0</VersionPrefix>
+    <VersionPrefix>3.1.0</VersionPrefix>
```

Comment:
Should be `3.0.0`, the current release version is `2.0.0`

Reply:
Same here. The current version of `Serilog.Extensions.Logging` package is `3.1.0`, I added this version as dependency. So why is the `Serilog.Extensions.Logging.File` package when using version `3.1.0` of base `Serilog.Extensions.Logging` should have version `3.0.0`? This is counter-intuitive for me.

Reply:
This package and Serilog.Extensions.Logging aren't coupled in that way. We won't (necessarily) revise this package when we update Serilog.Extensions.Logging, so they'll quickly get out of sync anyway

Reply:
OK, I see. Done.